src/Raven.Client/Constants.cs

```diff
@@ -597,6 +597,7 @@ internal static class VectorSearch
         {
             private const string EmbeddingPrefix = "embedding.";
 
+            internal const string EmbeddingForDocument = EmbeddingPrefix + "for";
```

Comment:
`embedding.for()` doesn't have any indication that it's going to accept a document ID as parameter. Maybe `embedding.forDoc()` ?